src/main/java/uk/ac/ebi/eva/pipeline/parameters/validation/ParametersValidatorUtil.java

```diff
@@ -22,12 +22,30 @@
 import java.nio.file.InvalidPathException;
 import java.nio.file.Path;
 import java.nio.file.Paths;
+import java.util.regex.Pattern;
 
 /**
  * Utility class to hold the low level checks on strings, dirs, files... parameters
  */
 public class ParametersValidatorUtil {
 
+    static void checkIsValidString(String stringToValidate,
+                                   String jobParametersName) throws JobParametersInvalidException {
+        checkIsNotNullOrEmptyString(stringToValidate, jobParametersName);
+        checkNonPrintableCharacters(stringToValidate, jobParametersName);
+        checkLength(stringToValidate, jobParametersName);
+    }
+
+    static void checkNonPrintableCharacters(String stringToValidate,
```

Comment:
`checkDoesNotContainPrintableCharacters` to match the full sentence structure used in the rest of the class.